tests/integration/test_postgresql_protocol/configs/users.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+ <yandex>
+     <profiles>
+         <default>
+         </default>
+     </profiles>
+
+     <users>
+         <default>
+             <password>123</password>
+             <profile>default</profile>
```

Comment:
The `<profile>` and `<quota>` lines are not necessary and could be deleted along with the `<profiles>` and `<quotas>` sections.

Reply:
Will be fixed.